src/spas/libs/member-management/feature/src/lib/member-management/user.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+import { User } from './user-profile';
+import { USERS } from './mock-users';
+import { Observable, of } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UserService {
```

Comment:
Create a user service interface and have `UserService` implement the interface so at build time we can search all mocks folder and switch out the "real" service with a mock service that will return mock data via an environment config setting like `ng b --configuration=mock`, something of that nature

Reply:
interface would look something like this:
```typescript
export interface UserInterface {
 getUsers: Observable<User[]>;
}
```
you would use it like this:

```typescript
export class UserService implements UserInterface {
 getUsers(): Observable<User[]> {
    return of(USERS);
  }
}
```